docs/upgrade-guides/upgrading-from-v4-to-v5.mdx

```diff
@@ -1196,6 +1214,35 @@ A couple of exported constants have been removed from `@clerk/nextjs` as you sho
     - `redirectToCreateOrganization` now returns `Promise<unknown>` instead of `void`
     - `redirectToHome` now returns `Promise<unknown>` instead of `void`
 
+- The `routing="path"` has become the default routing strategy for the following components and the `path` property is required for the path-based routing.
+
+  ```jsx
+    // Without path or routing props, an error with be thrown
+    <UserProfile />
+    <CreateOrganization />
+    <OrganizationProfile />
+    <SignIn />
+    <SignUp />
+    
+    // No prop example for SignIn / SignUp requires {framework_prefix}CLERK_SIGN_UP_URL / {framework_prefix}CLERK_SIGN_IN_URL env variable to be defined
+    <SignIn />
+    <SignUp />
+    
+    // Usage example #1
+    <UserProfile path="/whatever"/>
+    <CreateOrganization path="/whatever"/>
+    <OrganizationProfile path="/whatever"/>
+    <SignIn path="/whatever"/>
+    <SignUp path="/whatever"/>
+
+    // Usage example #2
+    <UserProfile routing="hash"/> // or routing="virtual"
+    <CreateOrganization routing="hash"/> // or routing="virtual"
+    <OrganizationProfile routing="hash"/> // or routing="virtual"
+    <SignIn routing="hash"/> // or routing="virtual"
+    <SignUp routing="hash"/> // or routing="virtual"
+  ```
```

Comment:
This change is applicable to `@clerk/nextjs`, `@clerk/remix`, `gatsby-plugin-clerk`, `@clerk/clerk-expo`,  `@clerk/chrome-extension` since those components are being re-exported.
cc: @LekoArts